### ISSUE TITLE
Traits in included files broke validation

### ISSUE THREAD

**Issue by nimblecrm** [raml-org/raml-js-parser-2#826]
Author of raml2html asked me to create the issue here.

When I use some of my traits in any of included files, it brokes raml2html validation. I attached the minimal example that is reproducing the problem.
```
Api contains errors.

INHERITING_UNKNOWN_TYPE: Inheriting from unknown type (resource.raml:5:8)
  ERROR_IN_INCLUDED_FILE: Error in the included file: Inheriting from unknown type (api.raml:16:2)
```

Commenting is part in resource.raml fixes it. Same for manual copy-pasting of resource.raml content.

[str.zip](https://github.com/raml-org/raml-js-parser-2/files/2380857/str.zip)

**Comment by KonstantinSviridov** [comment]
Thanks for you report, @nimblecrm  !

The fix will be released in 1.1.49

Regards,
Konstantin Sviridov

**Comment by KonstantinSviridov** [comment]
fixed by https://github.com/raml-org/raml-js-parser-2/commit/e2b533e658cba36fcd0fa4b8911bb400a52ffa4b
